ims/company/lost_or_stolen.md

```diff
@@ -0,0 +1,24 @@
+# What To Do When Your Device is Lost or Stolen
+
+The first thing you should do if your device is lost or stolen is report the incident to the CSO or IMS Manager as soon as possible.
```

Comment:
We should link to an explanation of CSO and IMS Manager as not everyone will find this page from the IMS README.